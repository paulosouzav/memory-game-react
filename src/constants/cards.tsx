import Asteroid from '../assets/cards/002-asteroid.png';
import Earth from '../assets/cards/006-planet-earth.png';
import Mars from '../assets/cards/017-mars.png';
import Sun from '../assets/cards/033-sun.png';
import Saturn from '../assets/cards/028-saturn.png';
import Spaceship from '../assets/cards/014-space-ship.png';
import DefaultImage from '../assets/cards/default-card.png';
import { ICard } from '../components/Card';
import { IImage } from '../components/Image';

export const DEFAULT_CARD_IMAGE: IImage = {
  src: DefaultImage,
  alt: 'Default',
};

export const CARDS: Array<ICard> = [
  {
    id: 'asteroid',
    image: {
      src: Asteroid,
      alt: 'Asteroid',
    },
  },
  {
    id: 'earth',
    image: {
      src: Earth,
      alt: 'Earth',
    },
  },
  {
    id: 'mars',
    image: {
      src: Mars,
      alt: 'Mars',
    },
  },
  {
    id: 'saturn',
    image: {
      src: Saturn,
      alt: 'Saturn',
    },
  },
  {
    id: 'spaceship',
    image: {
      src: Spaceship,
      alt: 'Spaceship',
    },
  },
  {
    id: 'sun',
    image: {
      src: Sun,
      alt: 'Sun',
    },
  },
];
