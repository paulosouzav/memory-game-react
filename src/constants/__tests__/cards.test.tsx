import { CARDS, DEFAULT_CARD_IMAGE } from '../cards';

describe('CARDS', () => {
  it('should have expected values', () => {
    expect(CARDS).toMatchSnapshot();
  });
});

describe('DEFAULT_CARD_IMAGE', () => {
  it('should have expected values', () => {
    expect(DEFAULT_CARD_IMAGE).toMatchSnapshot();
  });
});
