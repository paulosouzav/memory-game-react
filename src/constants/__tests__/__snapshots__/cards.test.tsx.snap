// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CARDS should have expected values 1`] = `
Array [
  Object {
    "id": "asteroid",
    "image": Object {
      "alt": "Asteroid",
      "src": "002-asteroid.png",
    },
  },
  Object {
    "id": "earth",
    "image": Object {
      "alt": "Earth",
      "src": "006-planet-earth.png",
    },
  },
  Object {
    "id": "mars",
    "image": Object {
      "alt": "Mars",
      "src": "017-mars.png",
    },
  },
  Object {
    "id": "saturn",
    "image": Object {
      "alt": "Saturn",
      "src": "028-saturn.png",
    },
  },
  Object {
    "id": "spaceship",
    "image": Object {
      "alt": "Spaceship",
      "src": "014-space-ship.png",
    },
  },
  Object {
    "id": "sun",
    "image": Object {
      "alt": "Sun",
      "src": "033-sun.png",
    },
  },
]
`;

exports[`DEFAULT_CARD_IMAGE should have expected values 1`] = `
Object {
  "alt": "Default",
  "src": "default-card.png",
}
`;
